subjects/smallest/README.md

```diff
@@ -0,0 +1,44 @@
+## smallest
+
+### Instructions
+
+Create a function named `smallest` that gets the smallest positive number in the `HashMap`.
```

Comment:
Here the fact we are limiting to "positive numbers" seems very confusing to me.
Why the function should return the smallest positive and not the smallest in general?
Also in that case the function name should change to reflect this specificity (something like smallest_positive).

I think the best is to get rid of this "positive" constraint so the function returns the smallest in general (ex. `[10, 22, -10, 4] -> `-10`).
Let me know if you do agree or if you'd prefer to keep this behavior :)

Reply:
I agree. Thanks for the suggestion.

Reply:
Done.